app/controllers/errors_controller.rb

```diff
@@ -1,6 +1,6 @@
 class ErrorsController < ApplicationController
   skip_load_and_authorize_resource only: %i[not_endpoint not_found internal_server_error dummy_exception]
-  protect_from_forgery prepend: true, except: %i[not_endpoint not_found internal_server_error]
+  skip_forgery_protection only: %i[not_endpoint not_found internal_server_error]
```

Comment:
check if default behaviour is `with: :null_session` rather than  `with: :exception` and the impact of this

Reply:
https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/4113/files#r669152854